operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -86,6 +86,27 @@ Use active voice in preference to passive voice. Active voice has the subject of
 
 Use the present tense.
   - For example: "Sharing this link lets other users view the linked message." as opposed to "Sharing this link will let other users view the linked message."
+  
+## Terminology Guide
+
+Throughout Mattermost, you’ll see the below words, terms are mentioned and used. Their definitions and reason for usage are defined below.
+
+| Terms          | Description        | Usage             |
+| ------------- |-------------| --------------------|
+| Mattermost | Used to refer to [Mattermost](https://mattermost.com/about-us/) the company and [Mattermost](https://mattermost.com/product/) the product. | <ul><li> Mattermost is an open-source, self-hostable online chat service with file sharing, search, and integrations</li> <li> Configure Mattermost on your server</li></ul> |
+| installation | Refers to an on-premises Mattermost environment comprising hardware (server/s), database, and filestore, etc which hosts the required files to run Mattermost. | <ul><li> Your Mattermost installation includes physical servers </li></ul> |
+| deployment | Used to refer to making a new installation widely accessible. It can also refer to releasing an updated version, patch, etc. | <ul><li> Desktop App Deployment Guide </li></ul> |
+| configuration | Used to describe the settings and customizations applied to Mattermost to change the appearance and behavior. | <ul><li> Edit the configuration file to include any custom settings related to your domain </li></ul> |
+| Mattermost Server | Used to refer to the single Linux binary that is installed on the physical server which hosts your Mattermost installation. It is not used to refer to the product going beyond the binary name. Always capitalize the "Server" when referring to the name of the binary. | <ul><li> Download the Mattermost Server binary |
+| Mattermost server | Used to refer to the physical server on which you run Mattermost. It is also referred to as the customer’s Mattermost server. Always lowercase "server" when referring to the customer’s physical Mattermost server | <ul><li> Your Mattermost server may need to be rebooted before changes are applied </li></ul> |
+| Web app | Used to refer to the Mattermost Web application. General web apps are not capitalized. | <ul><li> Mattermost Web app or Web app </li></ul> |
+| Mobile app | Used to refer to the Mattermost Mobile application. General mobile apps are not capitalized. | <ul><li> Mattermost Mobile app or Mobile app </li></ul> |
+| Desktop app | Used to refer to the Mattermost Desktop application. General desktop apps are not capitalized.| <ul><li> Mattermost Desktop app or Desktop app </li></ul> |
```

Comment:
Curious why "application" is in lower case. The name "Mattermost Desktop application" feels inconsistent with "Mattermost Server", compared to "Mattermost Desktop Application".

Definitely open to thoughts/feedback

Reply:
My take on it is:
"Desktop", "Mobile", "Web" are the names of the Mattermost product - as is "Server". So I wouldn't say "Mattermost Server Binary" and I wouldn't say "Mattermost Desktop Application" because "application" is a general application, not a specific Mattermost product. 

That said, there seem to be an equal number of "Web App" references as there are "Web app" in the documentation so I am 0/5 on which we use (for web, mobile, and desktop of course - just using web for illustrative purposes).